plugin-interface/src/main/scala/mesosphere/marathon/plugin/EnvVarValue.scala

```diff
@@ -0,0 +1,11 @@
+package mesosphere.marathon.plugin
+
+trait EnvVarValue
+
+trait EnvVarString extends EnvVarValue {
```

Comment:
Well, we want other plugins to use these types. For example, a the plugin
that generates DCOS_SECRETS_DIRECTIVE. Plugins don't depend on the core
marathon APIs -- it's the other way around.

On Wed, May 11, 2016 at 7:48 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In
> plugin-interface/src/main/scala/mesosphere/marathon/plugin/EnvVarValue.scala
> <https://github.com/mesosphere/marathon/pull/3856#discussion_r62833125>:
>
> > @@ -0,0 +1,11 @@
> > +package mesosphere.marathon.plugin
> > +
> > +trait EnvVarValue
> > +
> > +trait EnvVarString extends EnvVarValue {
>
> Why do we need these types here anyway? Also case classes can be extended
> if some plugin needs to.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/marathon/pull/3856/files/e24d8fc72711094a53f54eb5aef6d9d2fb073b52#r62833125>
>